src/packages/@ncigdc/modern_components/GenesBarChart/GenesBarChart.js

```diff
@@ -183,170 +184,54 @@ const Component = compose(
         onClick: () => handleClickGene(g, mutatedGenesChartData),
       }));
     const checkers = [
```

Comment:
maybe we can extract this hash map out since you're reusing it in the distribution bar chart too?

Reply:
Yes. Should do that.